google/cloud/audit/audit_log.proto

```diff
@@ -1,4 +1,4 @@
-// Copyright 2020 Google LLC
+// Copyright 2022 Google LLC
```

Comment:
nit: Is this the right date? I would have expected keeping the original or using 2024, but maybe the protos last changed in 2022?

Reply:
Fixed in https://github.com/googleapis/python-audit-log/pull/123/commits/0f26a6caa475845c0380c4b11c7477a1bba8f570